README.md

```diff
@@ -98,16 +99,6 @@ const options = [{ id: 'custom-id', value: 1, label: 'awesome' }];
 
 ### Customizing
 
-#### Styling with classnames
```

Comment:
i think we should still list the class names that a dev should target for each part.

Reply:
the class names table is re added and updated